common/lib/common-utils/src/bufferBrowser.ts

```diff
@@ -44,10 +44,29 @@ export const stringToBuffer = (input: string, encoding: string): ArrayBufferLike
 export const bufferToString = (blob: ArrayBufferLike, encoding: string): string =>
      IsoBuffer.from(blob).toString(encoding);
 
+/**
+ * Determines if an object is an array buffer
+ * @param obj - The object to determine if it is an ArrayBuffer
+ */
+export function isArrayBuffer(obj: any): obj is ArrayBuffer {
```

Comment:
Can we avoid this kind of check at all?

Reply:
No idea. @heliocliu , and @vladsud  have the most context here

Reply:
Yes we do.

Reply:
I believe that combination of these 2 PRs being in same release (i.e. compiled into both client and loader) should be sufficient to address the bug:
https://github.com/microsoft/FluidFramework/pull/5035
https://github.com/microsoft/FluidFramework/pull/5173

Later has exactly same error in description.
The problem is that we did not picked up first PR before cutting release.